bus-mapping/src/operation.rs

```diff
@@ -276,42 +278,36 @@ impl StorageOp {
         self.gc
     }
 
-    /// Returns the [`MemoryAddress`] associated to this Operation.
-    pub const fn address(&self) -> &MemoryAddress {
-        &self.addr
+    /// Returns the [`EthAddress`] corresponding to this storage operation.
+    pub const fn address(&self) -> &EthAddress {
+        &self.address
+    }
+
+    /// Returns the [`EvmWord`] used as key for this operation.
+    pub const fn key(&self) -> &EvmWord {
+        &self.key
     }
 
     /// Returns the [`EvmWord`] read or written by this operation.
     pub const fn value(&self) -> &EvmWord {
         &self.value
     }
 
-    /// Returns the [`EvmWord`] read or written by previous operation.
+    /// Returns the [`EvmWord`] at key found previous to this operation.
     pub const fn value_prev(&self) -> &EvmWord {
         &self.value_prev
     }
-
-    /// Returns the underlying value representation.
-    pub const fn storage_key(&self) -> &EvmWord {
-        &self.storage_key
-    }
 }
 
 impl PartialOrd for StorageOp {
     fn partial_cmp(&self, other: &StorageOp) -> Option<Ordering> {
-        match self.address().partial_cmp(other.address()) {
-            None => None,
-            Some(ord) => match ord {
-                std::cmp::Ordering::Equal => self.gc().partial_cmp(&other.gc()),
-                _ => Some(ord),
-            },
-        }
+        Some(self.cmp(other))
     }
 }
 
 impl Ord for StorageOp {
     fn cmp(&self, other: &StorageOp) -> Ordering {
-        match self.address().cmp(other.address()) {
+        match self.key().cmp(other.key()) {
             Ordering::Equal => self.gc().cmp(&other.gc()),
             Ordering::Less => Ordering::Less,
             Ordering::Greater => Ordering::Greater,
```

Comment:
`address` should be the first separator for `StorageOp`

```suggestion
        match self.address().cmp(other.address()) {
            Ordering::Equal => match self.key().cmp(other.key()) {
                Ordering::Equal => self.gc().cmp(&other.gc()),
                Ordering::Less => Ordering::Less,
                Ordering::Greater => Ordering::Greater,
            },
            Ordering::Less => Ordering::Less,
            Ordering::Greater => Ordering::Greater,
```

Reply:
ah sorry! I changed the ordering of StorageOp when I deleted the address, and I forgot to leave it as before.  I've updated the PR.